### ISSUE TITLE
Auto Completion in Search

### ISSUE THREAD

**Issue by tsah20** [OpenGenus/quark#21]
**This is a(n):**

- [X] New feature proposal
- [ ] Reporting a keyword
- [ ] Error
- [X] Proposal to the Search Engine
- [ ] Other

**Details:**

Currently there is no auto-completion in the search box to guide the user. If this enhances the product, we would like to work on it.

**Comment by tsah20** [comment]
@AdiChat  Please review the suggestion

**Comment by AdiChat** [comment]
Yes, this will be a useful feature and result in a guided learning experience. It looks like there exists a pull request regarding this feature. :+1:
